src/Domain/Creators/Sphere.cpp

```diff
@@ -170,9 +170,7 @@ Sphere::Sphere(
       const std::string wedge_name =
           shell_prefix + gsl::at(wedge_directions, direction);
       block_names_.emplace_back(wedge_name);
-      if (num_shells_ > 1) {
```

Comment:
FYI, this wasn't included because `Shell0` doesn't really make sense if there's only one shell. The group is identical to the `Wedges` group (which holds all shells). My weak preference is to leave this as-is, or name the group `Shell` (without the index).

Reply:
ok fine with me